desktop/public_desktop/about.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
```

Comment:
Could we use the [main public location](https://github.com/Automattic/wp-calypso/tree/master/public) for these?